<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1473756853658" stop="1473757132639" version="1.4.22" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Default Suite : test</name>
    <title>Default Suite : test</title>
    <test-cases>
        <test-case start="1473756853694" stop="1473756853697" status="pending">
            <name>step7_assertValidationOnUpdateClosedHistory</name>
            <description>Проверка валидации на изменение закрытой ИБ </description>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="20572@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1473756865996" stop="1473756998542" status="passed">
            <name>beforeTest</name>
            <description>Подготовка тестовых данных: удаление диагнозов пациента, если такие существуют</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="20572@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1473756998591" stop="1473757003211" status="passed">
            <name>step1_openPage</name>
            <description>Отркытие страницы Заведующий отделением</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="20572@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1473757003217" stop="1473757023271" status="passed">
            <name>step2_openMedicalHistory</name>
            <description>Открытие истории болезни</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="20572@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1473757023277" stop="1473757056060" status="passed">
            <name>step3_createDiagnosis</name>
            <description>Создание диагноза</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="20572@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1473757056065" stop="1473757099358" status="passed">
            <name>step4_editDiagnosis</name>
            <description>Редактирование диагноза</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="20572@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1473757099362" stop="1473757127758" status="failed">
            <name>step5_discharge</name>
            <description>Выписать</description>
            <failure>
                <message>AssertionError: Cannot find value '1 Первичная медико-санитарная помощь' in combobox careLevel.</message>
                <stack-trace>java.lang.AssertionError: Cannot find value '1 Первичная медико-санитарная помощь' in combobox careLevel.
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
	at net.framework.autotesting.meta.components.Classifier.searchValueInCombobox(Classifier.java:332)
	at net.framework.autotesting.meta.components.Classifier.setValue(Classifier.java:357)
	at businessFramework.module.hospital.modals.discharge.DischargeModal.fillDischargeForm(DischargeModal.java:33)
	at businessFramework.module.hospital.pages.medicalHistory.MedicalHistoryPage.discharge(MedicalHistoryPage.java:466)
	at test.businessTests.hospital.MedicalHistoryTest.step5_discharge(MedicalHistoryTest.java:151)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="20572@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1473757127760" stop="1473757127762" status="canceled">
            <name>step6_editDischarge</name>
            <description>Редактировть выписку</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:356)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:171)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1887)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="20572@KZN-062.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1473757127763" stop="1473757132635" status="failed">
            <name>step8_cancelDischarge</name>
            <description>Отмена выписки и проверка подтверждения очищения данных о смерти</description>
            <failure>
                <message>AssertionError: Error when click on element [[[[[[FirefoxDriver: firefox on WINDOWS (16fdd2ec-51f8-457c-92d1-3c2ce9b58f4d)] -&gt; tag name: body]] -&gt; css selector: div[id='n2o:hospitalRecordList'][class*='n2o-container']]] -&gt; css selector: *[data-action-id='reopen']]</message>
                <stack-trace>java.lang.AssertionError: Error when click on element [[[[[[FirefoxDriver: firefox on WINDOWS (16fdd2ec-51f8-457c-92d1-3c2ce9b58f4d)] -&gt; tag name: body]] -&gt; css selector: div[id='n2o:hospitalRecordList'][class*='n2o-container']]] -&gt; css selector: *[data-action-id='reopen']]
	at org.testng.Assert.fail(Assert.java:83)
	at net.framework.autotesting.base.BaseHelperWithWebDriver.click(BaseHelperWithWebDriver.java:167)
	at net.framework.autotesting.meta.components.Button.click(Button.java:72)
	at businessFramework.module.hospital.pages.medicalHistory.MedicalHistoryPage.clickCancelDischarge(MedicalHistoryPage.java:444)
	at businessFramework.module.hospital.pages.medicalHistory.MedicalHistoryPage.cancelDischargeWithClearDeathDataAssertion(MedicalHistoryPage.java:486)
	at test.businessTests.hospital.MedicalHistoryTest.step8_cancelDischarge(MedicalHistoryTest.java:182)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: org.openqa.selenium.ElementNotVisibleException: Element is not currently visible and so may not be interacted with
Command duration or timeout: 34 milliseconds
Build info: version: '2.51.0', revision: '1af067dbcaedd7d2ab9af5151fc471d363d97193', time: '2016-02-05 11:20:57'
System info: host: 'KZN-062', ip: '10.77.104.134', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_102'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=46.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 16fdd2ec-51f8-457c-92d1-3c2ce9b58f4d
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:327)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:85)
	at net.framework.autotesting.base.BaseHelperWithWebDriver.click(BaseHelperWithWebDriver.java:162)
	... 32 more
Caused by: org.openqa.selenium.ElementNotVisibleException: Element is not currently visible and so may not be interacted with
Build info: version: '2.51.0', revision: '1af067dbcaedd7d2ab9af5151fc471d363d97193', time: '2016-02-05 11:20:57'
System info: host: 'KZN-062', ip: '10.77.104.134', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_102'
Driver info: driver.version: unknown
	at &lt;anonymous class&gt;.fxdriver.preconditions.visible(file:///C:/Users/BULAT~1.GAR/AppData/Local/Temp/anonymous6605584214757668528webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:10045)
	at &lt;anonymous class&gt;.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/BULAT~1.GAR/AppData/Local/Temp/anonymous6605584214757668528webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12585)
	at &lt;anonymous class&gt;.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/BULAT~1.GAR/AppData/Local/Temp/anonymous6605584214757668528webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12602)
	at &lt;anonymous class&gt;.DelayedCommand.prototype.executeInternal_(file:///C:/Users/BULAT~1.GAR/AppData/Local/Temp/anonymous6605584214757668528webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12607)
	at &lt;anonymous class&gt;.DelayedCommand.prototype.execute/&lt;(file:///C:/Users/BULAT~1.GAR/AppData/Local/Temp/anonymous6605584214757668528webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12549)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="KZN-062"/>
                <label name="thread" value="20572@KZN-062.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
